### PR TITLE
Dev chapter4 5

### PR DESCRIPTION
**Pull request by liliamyhaliuk**
I did exercises from chapters 4 and 5 of the book "Think Python". Please, check when you have time.

### CODE REVIEW

File: address_book/class_Contact.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""
```
Comment (auraz) [inline address_book/class_Contact.py:1]: Please remove from here (git -rm filename) and commit

File: address_book/main_menu.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
+"""
```
Comment (auraz) [inline address_book/main_menu.py:1]: please remove from here and put in the relevant repo

**Review by auraz — APPROVED** [review]
Excellent!

File: address_book/class_Contact.py (hunk 1 above)
Comment (liliamyhaliuk) [reply address_book/class_Contact.py]: I deleted irrelevant files. Thank you!

File: address_book/main_menu.py (hunk 2 above)
Comment (liliamyhaliuk) [reply address_book/main_menu.py]: I deleted irrelevant files. Thank you!

**Pull request merged**